src/types/express/session.d.ts

```diff
@@ -0,0 +1,8 @@
+export interface SessionData {
+  userInfo: {
+    accessToken?: string
+    githubId?: string
+    isomerUserId: string
+    email: string
```

Comment:
double checking - for a github user, what is their `isomerUserId`? is it just `githubId`?

Reply:
nope - github users also have a db entry in the users table, and the id in that table is used